src/main/kotlin/com/mdrobnak/lalrpop/inspections/MissingTerminalsInspection.kt

```diff
@@ -0,0 +1,57 @@
+package com.mdrobnak.lalrpop.inspections
+
+import com.intellij.codeInspection.LocalInspectionTool
+import com.intellij.codeInspection.LocalInspectionToolSession
+import com.intellij.codeInspection.ProblemHighlightType
+import com.intellij.codeInspection.ProblemsHolder
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiElementVisitor
+import com.mdrobnak.lalrpop.psi.*
+
+object MissingTerminalsInspection : LocalInspectionTool() {
```

Comment:
This works great when you use `match` in your grammar, but it fails when there is no match block. It should only flag missing terminals if there exists a `match` block.

Additionally, if there is an underscore in the `match` block (specifically at the end, but don't worry about enforcing that) then it acts as a catch-all, so there can't be unresolved terminals.

Reply:
> but it fails when there is no match block.

Now I'm really interested in how would a lalrpop grammar look like if that were the case(also remove the enum in the extern block). Would lalrpop even be able to make sense out of it?